cli/flox/src/commands/environment.rs

```diff
@@ -507,16 +516,28 @@ pub enum ListMode {
     /// Show the raw contents of the manifest
     #[bpaf(long, short)]
     Config,
```

Comment:
question blocking: can we drop `Config`? Is having that deliberate or just leftover for some reason?

Reply:
That's deliberate and asked for

Reply:
Its to check the contents of remote envs prior to pulling and activating them as a security measure